modules/quadbin/bigquery/sql/QUADBIN_BOUNDARY.sql

```diff
@@ -0,0 +1,42 @@
+----------------------------
+-- Copyright (C) 2022 CARTO
+----------------------------
+
+CREATE OR REPLACE FUNCTION `@@BQ_PREFIX@@carto.QUADBIN_BOUNDARY`(quadbin INT64)
+RETURNS GEOGRAPHY
+AS (
+    CASE
+        WHEN quadbin IS NULL THEN
+            NULL
+        -- Deal with level 0 boundary issue.
+        WHEN quadbin=5192650370358181887 THEN
+            ST_GEOGFROMGEOJSON('{"coordinates":[[[-180,85.0511287798066],[-180,-85.0511287798066],[180,-85.0511287798066],[180,85.0511287798066],[-180,85.0511287798066]]],"type":"Polygon"}')
```

Comment:
Why at level 0 are we letting BigQuery to compute the spherical coordinates but at level 1 we are returning the planar polygon? `ST_GEOGFROMTEXT` does not compute spherical coordinates

Reply:
From https://github.com/CartoDB/analytics-toolbox-core/pull/176. BigQuery uses by default spherical coordinates, but GeoJSON forces BigQuery to density the edges and make them look planar. If we use WKT for level 0, the expression generates a line, because -180 and 180 are the same point and it's simplified.

```sql
SELECT ST_GEOGFROMTEXT('POLYGON((-180 85.0511287798066, 180 85.0511287798066, 180 -85.0511287798066, -180 -85.0511287798066, -180 85.0511287798066))')
-- LINESTRING(-180 85.0511287798066, 180 0, -180 -85.0511287798066)
```

Reply:
Yes, I know, I mean the other way around. We're not densifying level 1 (Use also GeoJSON for level 1)

Reply:
It can be done with `planar => TRUE`, but I think we should try to avoid densifying, although it's more precise defining planar edges for the grid would have a penalty in performance and storage, and it happens the same with all levels although the distortion is reduced with zoom level. We can do deeper research with numeric metrics in the future